lib/options.ts

```diff
@@ -23,6 +23,12 @@ export interface Options {
    * @default - python is not generated
    */
   python?: PythonOutputOptions;
+
+  /**
+   * Produce java code.
+   * @default - java is not generated
```

Comment:
Add a note here that says that the code will be produced under `src` (with an example of the full path combined with the package name). Also mention the requirements from `Pom.xml` (dependency on jsii runtime and resource).